### PR TITLE
fix(webapp): bump lexical and fix preview-off serialization [WPB-22420]

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-22420" title="WPB-22420" target="_blank">TaskWPB-22420</a>  [Web] General maintenance ticket for PR merges
  </td></table>
  <br />
 

Upgrade Lexical to 0.41.0 and stop using Lexical markdown export when markdown preview is disabled.

This restores correct sending of raw markdown input, fixes extra blank lines in multiline messages, and keeps mention offsets aligned in preview-off mode.

Adds regression tests for serializer and multiline mention edge cases.

### CODE REVIEW

File: apps/webapp/src/script/components/InputBar/InputBarEditor/RichTextEditor/utils/serializeMessage.ts [hunk 1]
```diff
@@ -24,7 +24,10 @@ import {markdownTransformers} from './markdownTransformers';
 
 export const serializeMessage = (showMarkdownPreview: boolean): string => {
   if (!showMarkdownPreview) {
-    return $getRoot().getTextContent();
+    return $getRoot()
```
Comment (arjita-mitra) [inline apps/webapp/src/script/components/InputBar/InputBarEditor/RichTextEditor/utils/serializeMessage.ts:27]: Can you confirm:
- this fix correctly handle cases where a node is an empty paragraph (a blank line the user deliberately added)?
- Are there test cases covering: multiline text with mentions, multiline with mixed formatting (bold + italic in different lines), and intentional blank lines between content?

Comment (arjita-mitra) [reply apps/webapp/src/script/components/InputBar/InputBarEditor/RichTextEditor/utils/serializeMessage.ts:27]: @thisisamir98 will take care of this in next PR. therefore approving.

**Review by arjita-mitra — APPROVED** [review]

**Review by e-maad — APPROVED** [review]
